pkg/apis/targets/v1alpha1/googlecloudpubsubtarget_lifecycle.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+
+	"github.com/triggermesh/triggermesh/pkg/apis/common/v1alpha1"
+)
+
+// Managed event types
+const (
+	EventTypeGoogleGenericResponse = "io.triggermesh.google.pubsub.response"
```

Comment:
Probably it's better to name it something like `GoogleCloudPubSubResponseEventType` because otherwise, this constant seems to be common for all Google Cloud resources.